Core/Libraries/Source/WWVegas/WWLib/win.h

```diff
@@ -69,7 +71,6 @@
 #pragma warning(pop)
 #endif
 
-#ifdef _WINDOWS
```

Comment:
Is this move correct? This define comes from windows.h, no? Perhaps there needs to be a new `#ifdef _WIN32` higher up?